src/llmcompressor/transformers/compression/sparsity_config.py

```diff
@@ -82,6 +90,11 @@ def from_pretrained(
         :param state_dict: optional state_dict to replace that in model, used for
         gathering global FSDP model info
         :param compress: whether or not to compress the model on disk
+        :param quantization_format: the quantization compression format being used
+            for the model
+        :param no_sparse_compression: whether or not to compress the model with
+            sparse compressors, If True, the sparse compression format will
+            be dense, default is False.
```

Comment:
Don't we want the default behavior to be dense format for the widest compatibility? Just noting that this seemingly changes the default behavior for existing 2:4 quant flows

Reply:
wdym by default existing 2:4 quant flows?